meeko_temp_task_refinement/MOLSETUP_NOTES.md

```diff
@@ -0,0 +1,92 @@
+# Molecule Setup Notes
+
+This is a document meant as a temporary stop-gap to help us get technical tasks organized.
+This one pertains specifically to molecule setups, and contains notes, bugs, and feature 
+requests from the original molsetup.py file as well as details for plans to further update
+and refactor molecule setups, and information about the status of what has been done so far.
+
+### Implementation Notes From The Original File
+
+* Modify so there are no more dictionaries and only lists/arrays.
+  * methods like `add_x`, `delete_x`, `get_x` wil deal with indexing
+* Provide infrastructure to calculate "differential"
+modifications, like tautomers: the class calculating tautomers will copy and modify the setup 
+to swap protons and change the atom types (and possibly rotations?), then
+there will e a function to extract only differences (i.e., atom types and coordinates,
+in the case of tautomers) and store them to make it a switch-state
+* Only use 1-dimensional arrays (C)
+* Update `add_atom` to not specify neighbors, which should be defined only when a bond is created?
```

Comment:
What did this mean/ is this something I should cover as we're making these changes

Reply:
It has been fixed meanwhile, so nothing left to do anymore. Not sure when it was fixed. In v3.0.0 there was still a `neighbors` argument in `add_atom`: https://github.com/forlilab/Meeko/blob/26e9ba24bdccd3e8ad766201b7d6d9d3a1a916ec/meeko/setup.py#L145